engines/wintermute/base/gfx/opengl/base_render_opengl3d.cpp

```diff
@@ -189,6 +189,7 @@ bool BaseRenderOpenGL3D::setProjection(float fov) {
 	glMatrixMode(GL_PROJECTION);
 	glLoadIdentity();
 	glFrustum(-top * aspectRatio, top * aspectRatio, -top, top, nearPlane, farPlane);
+	glGetFloatv(GL_PROJECTION_MATRIX, _lastProjectioMatrix.getData());
```

Comment:
Typo, missing an n in _lastProjectionMatrix

Reply:
please correct typo